cmd/notation/cert/delete.go

```diff
@@ -33,6 +33,17 @@ func certDeleteCommand(opts *certDeleteOpts) *cobra.Command {
 			}
 			return nil
 		},
+		Long: `Delete certificates from the trust store
+
+Example - Delete all certificates with "ca" type from the trust store "acme-rockets":
+  notation certdelete --type ca --store "acme-rockets" --all
```

Comment:
missing space between `notation cert` and `delete`

Reply:
Fixed